op/opstatus.go

```diff
@@ -140,6 +143,35 @@ func (s *Status) Check() HealthResult {
 	return hr
 }
 
+// AddCheckerWithMetrics adds a function that can check the applications health and meter the outcomes of the checks
+// Multiple checkers are allowed.  The checker functions should be capable of
+// being called concurrently (with each other and with themselves).
+func (s *Status) AddCheckerWithMetrics(name string, checkerFunc func(cr *CheckResponse)) *Status {
```

Comment:
Would it make sense to standardize this across UW and have just one metric behind health status rather than individuals metrics for each check registered by the app?

So currently we would end up with something like:
```
mongo_check{healthcheck_result="healthy"} 1
kafka_check{healthcheck_result="healthy"} 1
```

I'm proposing having something like:

```
health_status{healthcheck_name="mongo_check" healthcheck_result="healthy"} 1
health_status{healthcheck_name="kafka_check" healthcheck_result="healthy"} 1
```
This would result in reusable dashboards and would also simplify aggregations against these metrics.

Reply:
That would be my preferred approach too, in fact it's what the javascript implementation does already.

Reply:
Sounds like a much better to aggregate if done like what you are suggesting. Will make these changes.